src/components/template/FeedTemplate/styled.jsx

```diff
@@ -0,0 +1,16 @@
+import styled, { css } from 'styled-components';
+
+export const FeedWrapper = styled.section`
+  ${({ theme }) => {
+    return css`
+      width: 600px;
+      margin: 0 auto;
+      border: 1px solid black;
```

Comment:
```suggestion
      border: 1px solid black;
```
- 저희가 블랙 색상도 사용 했었군요!! theme에 추가해주시고 추가한 걸로 작성하시면 좋을 것 같습니다ㅎㅎ

Reply:
이...이것은... 저의 실수에요....ㅎㅎㅎㅎ 
맨~처음 스타일링할 때 피드 전체 wrapper 부분에 크기가 얼만큼인지 명시해두고 작업하려고 적용했던건데 메인 배경색이 어둡다보니 티가 안나서 마지막에 삭제하는 것을 깜박했습니다 🥲 
다음부터는 주의하겠습니다 수정 완료했습니다!